recipes/serial/all/conanfile.py

```diff
@@ -35,6 +45,8 @@ def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
+        if self.settings.os == "Windows" and self.options.shared:
+            self._cmake.definitions["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
```

Comment:
We tend to keep this in the `cmake_wrapper` if you can please move it

Reply:
Sure! I guess I just happened to pick the wrong way :)